script.js

```diff
@@ -0,0 +1,45 @@
+// Set variables for each respective move
+var x = "X"
+var o = "O"
+// Create functions to place X or O, taking in boxNumber to choose where to place
+function placeX(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = x;
+}
+function placeO(boxNumber) {
+	document.getElementById(boxNumber).innerHTML = o;
+}
+
+// Set variables to easily select individual cells and determine which player's turn it is
+
+var cells = document.getElementsByTagName("TD");
```

Comment:
Better to use a class name that's on each TD cell, the reason being, if you later add another table to the HTML (unrelated to the game board), you don't want to inadvertently select all of those as well here.